packages/line/src/Mesh.js

```diff
@@ -68,6 +72,41 @@ const Mesh = ({
         [onClick]
     )
 
+    const handleTouchStart = useCallback(
```

Comment:
`showTooltipAt`, `tooltip`, `margin` and `setCurrent` dependencies are missing (same for `handleTouchMove`).

Reply:
Looks like the were some missing deps on the mouse ones too, patched in https://github.com/WilliamABradley/nivo/commit/f2e5fbb6007e02bbf0a7ea0f4f363916fe7ee138